Contents/mods/CommunityAPI/media/lua/client/ModSettingAPI/ModSetKeyBindDialog.lua

```diff
@@ -0,0 +1,66 @@
+
+---@class ModSetKeyBindDialog
+ModSetKeyBindDialog = ISPanel:derive("ModSetKeyBindDialog")
```

Comment:
Having object this this can be set to local and returned at the end of the file.
Then can be required into other file like this
```lua
local ModSetKeyBindDialog = require("ModSettingAPI/ModSetKeyBindDialog")
```